packages/peregrine/lib/talons/CartPage/ProductListing/EditModal/useProductForm.js

```diff
@@ -18,26 +18,44 @@ export const useProductForm = props => {
     const [{ cartId }] = useCartContext();
 
     const [optionSelections, setOptionSelections] = useState(new Map());
-    const [formApi, setFormApi] = useState();
 
     const [
         updateItemQuantity,
-        { called: updateQuantityCalled, loading: updateQuantityLoading }
+        {
+            called: updateQuantityCalled,
+            error: updateQuantityError,
+            loading: updateQuantityLoading
+        }
     ] = useMutation(updateQuantityMutation);
     const [
         updateConfigurableOptions,
-        { called: updateConfigurableCalled, loading: updateConfigurableLoading }
+        {
+            called: updateConfigurableCalled,
+            error: updateConfigurableError,
+            loading: updateConfigurableLoading
+        }
     ] = useMutation(updateConfigurableOptionsMutation);
 
     const isSaving =
         (updateQuantityCalled && updateQuantityLoading) ||
         (updateConfigurableCalled && updateConfigurableLoading);
 
-    useEffect(() => {
-        if (formApi) {
-            formApi.setValue('quantity', cartItem.quantity);
```

Comment:
What was this doing and why is it gone now?

Reply:
The intention was to make sure quantity was updated between renders, but two things made it obsolete:

1. I started un-mounting between renders, so this wasn't needed
2. This was fixed in the Quantity component in the same way, so this was redundant anyway (even if we didn't unmount between renders)